Dockerfile

```diff
@@ -1,8 +1,29 @@
 # Get ALPINE image
 FROM alpine:3.14
 
-# Download DEBIAN packages
-RUN apk update && apk add --no-cache bash curl git openssl openssh jq
+# Install packages
+RUN apk update && apk add --no-cache bash curl git openssl openssh jq openrc libgcc gcompat
+
+# install bun
+RUN curl -fsSL https://bun.sh/install | bash
```

Comment:
Do you think **Bun** is better than **NPM**  or vice versa? I haven't used **Bun** before so my question is what are the best use cases to use **Bun**?

Reply:
Bun binaries are much smaller. 
Bun is faster. 
Bun uses far less ram

There could be problems with bun as it's less tested than Nodejs. There was a time things were not supported, but at this point it's about on par with nodejs supporting all APIs. So I use it if it works.